smoke-test/metric_service_smoke_test.js

```diff
@@ -16,23 +16,24 @@
 
 describe('MetricServiceSmokeTest', () => {
   if (!process.env.GCLOUD_PROJECT) {
-    throw new Error("Usage: GCLOUD_PROJECT=<project_id> node #{$0}");
+    throw new Error('Usage: GCLOUD_PROJECT=<project_id> node #{$0}');
   }
-  var projectId = process.env.GCLOUD_PROJECT;
+  let projectId = process.env.GCLOUD_PROJECT;
 
   it('successfully makes a call to the service using promises', done => {
     const monitoring = require('../src');
 
-    var client = new monitoring.v3.MetricServiceClient({
+    let client = new monitoring.v3.MetricServiceClient({
```

Comment:
client is never reassigned within block, use `const`?

Reply:
Right now these are automated changes that will be fixed when we enable `prefer-const` in the .eslintrc.  I'm tired of leaving this feedback too so I'm enabling a lint rule :P